src/sections/DonationDetailSection.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import { Box, Heading, Text, Stack, Center, Grid } from '@chakra-ui/react';
+import { graphql } from 'gatsby';
+import Img from 'gatsby-image';
+import SectionHeader from 'src/components/SectionHeader';
+
+interface Props {
+  data: GatsbyTypes.DonationDetailFragment;
+}
+
+function DonationDetailSection({ data }: Props): JSX.Element {
+  const section = data.contentfulDonationDetailSection;
+
+  return (
+    <Box bg="gray.extralight">
+      <SectionHeader text="Where do donations go" textPosition="left" />
+      <Grid bg="charcoal" templateColumns={{ base: '1fr', lg: '1fr 1fr' }}>
+        <Box h="800px" w="full">
```

Comment:
The text might grow larger than 800px so the image becomes too short: 
![image](https://user-images.githubusercontent.com/8410924/113704517-3b64ca80-96aa-11eb-93ae-d8167b5b46b9.png)